Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -391,9 +391,8 @@ where typeof(ITrackable).IsAssignableFrom(prop.PropertyType)
             // Recursively populate related entities on ref and child properties
             foreach (var item in items)
             {
-                // Avoid endless recursion
-                if (!visitationHelper.TryVisit(item)) continue;
-
+                // Avoid loading related entities of complext types, and avoid endless recursion
+                if (!visitationHelper.TryVisit(item) || IsComplexType(context, item.GetType())) continue;
```

Comment:
You need to make this change also to the async version, `LoadRelatedEntitiesAsync`.